vsphere/manifest.json

```diff
@@ -46,7 +46,8 @@
     },
     "dashboards": {
       "vsphere-overview": "assets/dashboards/vsphere_overview.json",
-      "VMware vSphere TKG - Overview": "assets/dashboards/vmware_vsphere_tkg_-_overview.json"
+      "VMware vSphere TKG - Overview": "assets/dashboards/vmware_vsphere_tkg_-_overview.json",
+      "VMware vSphere- Property Metrics": "assets/dashboards/vmware_vsphere-_property_metrics.json"
```

Comment:
noob question and probably nit comment, is the `-` part of the vSphere product name (`vSphere-`) or is it supposed to be a separator (`VMware vSphere - Property Metrics`) similar to the other dashboard we already have: `VMware vSphere TKG - Overview`?

Reply:
+1

Reply:
It's supposed to be a separator similar to the other dashboards!